app/sdk/controllers/DashBoardController.java

```diff
@@ -0,0 +1,92 @@
+package sdk.controllers;
+
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.google.inject.Inject;
+import play.libs.Json;
+import play.libs.ws.WSClient;
+import play.libs.ws.WSRequest;
+import play.mvc.Controller;
+import play.mvc.Http;
+import play.mvc.Result;
+import play.mvc.With;
+import sdk.AppTree;
+import sdk.ValidateRequestAction;
+import sdk.dashboard.DashBoard;
+import sdk.datasources.DashBoardSource_Internal;
+import sdk.utils.*;
+
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CompletionStage;
+
+import static sdk.utils.CallbackLogger.logCallbackInfo;
+import static sdk.utils.CallbackLogger.logExceptionCallback;
+
+public class DashBoardController extends Controller {
+    @Inject
+    protected WSClient wsClient;
+
+    @With({ValidateRequestAction.class})
```

Comment:
You're going to need to temporarily remove this ValidateRequestAction. The web client will not have the connector API needed for this. Once this is routed through the core we can add it back.